node_modules/oae-principals/emailTemplates/invite.txt.jst

```diff
@@ -0,0 +1,23 @@
+<%
+    var greeting = util.i18n.translate('__MSG__ACTIVITY_EMAIL_GREETING__', {'displayName': util.html.encodeForHTML(user.displayName)});
+    greeting = util.html.toText(greeting, false);
+%>
+
+<%= greeting %>
+
+<%= inviter.displayName %> wants to collaborate with you. You can do so by visiting the following URL and creating an account.<br />
```

Comment:
This all needs to become internationalisable. I'm assuming it's waiting for the real designed template implementation as well?